src/prpy/planning/vectorfield.py

```diff
@@ -95,8 +95,11 @@ def PlanToEndEffectorPose(self, robot, goal_pose, timelimit=5.0,
         manip = robot.GetActiveManipulator()
 
         def vf_geodesic():
+            """
+            Function defining a joint-space vector field.
+            """
```

Comment:
This `__docstring__` is misleading. An *arbitrary* vector field function implements a joint-space vector field. This function specifically implements a vector field that moves along the geodesic towards `goal_pose`.

I also don't think this function strictly needs a `__doscstring__` - although I'm certainly not going to argue with adding one - since it is only accessible inside this function.